$NetBSD$

Move all fgallery files into view subdirectory so we can symlink it on
webserver to /usr/pkg/share/fgallery/view in DocumentRoot hierarchies

--- view/index.html.orig	2016-01-06 19:45:03.000000000 +0000
+++ view/index.html
@@ -2,18 +2,16 @@
 <html>
   <head>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-    <meta name="viewport" content="width=device-width,initial-scale=1" />
-    <script src="mootools-core-1.4.js" type="text/javascript"></script>
-    <script src="mootools-more-1.4.js" type="text/javascript"></script>
-    <script src="mootools-idle.js" type="text/javascript"></script>
-    <script src="mootools-mooswipe.js" type="text/javascript"></script>
-    <script src="index.js" type="text/javascript"></script>
-    <link href="index.css" rel="stylesheet" type="text/css"/>
-  </head>
-  <body>
+    <meta name="viewport" content="width=device-width,initial-scale=1,minimum-scale=1,minimal-ui" />
     <noscript>
-      <h2>Frak! JavaScript required :'(</h2>
+      <meta http-equiv="refresh" content="0;url=noscript.html" />
     </noscript>
-    <div id="gallery"></div>
-  </body>
+    <script src="view/mootools-core-1.4.js" type="text/javascript"></script>
+    <script src="view/mootools-more-1.4.js" type="text/javascript"></script>
+    <script src="view/mootools-idle.js" type="text/javascript"></script>
+    <script src="view/mootools-mooswipe.js" type="text/javascript"></script>
+    <script src="view/index.js" type="text/javascript"></script>
+    <link href="view/index.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body><div id="gallery"></div></body>
 </html>
